Editor/ApplyOnPlay.cs

```diff
@@ -123,6 +123,13 @@ private static void RecreateAnimators(Transform avatar)
             // it can still start moving around stale bone references.
             var tmpObject = new GameObject();
             
+#if CVR_CCK_EXISTS
+            // ChilloutVR is not a package, and does not have an assembly definition.
+            var t_CVRAvatar = AppDomain.CurrentDomain.GetAssemblies()
+                .SelectMany(assembly => assembly.GetTypes())
+                .FirstOrDefault(t => t.FullName == "ABI.CCK.Components.CVRAvatar");
```

Comment:
Should this be generalized to components that depend on Animator in general? That would avoid NDMF core needing to know about CVR CCK (and thus make writing a test for this behavior much easier)

Reply:
You want to enumerate the sibling components on the Animator and find out which ones have the RequireComponent attribute?

Reply:
Addressing in 85a213c